packages/modeling/src/curves/bezier/tangentAt.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Calculates the position at a specific point along a bezier easing curve.
```

Comment:
Umm... it seems that the documention has been reversed in tangetAt and valueAt.

Reply:
hah!  thanks